cloud_experts_tutorials/cloud-experts-rosa-osd-change-default-domain.adoc

```diff
@@ -27,6 +29,7 @@ The changes that we make to the component routes{fn-term-component-routes} in th
 * link:https://docs.openshift.com/container-platform/latest/web_console/customizing-the-web-console.html#customizing-the-console-route_customizing-web-console[Customizing the console route]
 * link:https://docs.openshift.com/container-platform/latest/web_console/customizing-the-web-console.html#customizing-the-download-route_customizing-web-console[Customizing the download route]
 
+[id="prerequisites_{context}"]
```

Comment:
This will work fine, just FYI-for-future that there is a repo guidelines section about specifically [Prereqs/Addtl Resources/Next Steps in assemblies](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies) (tho maybe less important for ROSA, idk).